recipes/gmail/webview.js

```diff
@@ -25,14 +25,23 @@ module.exports = Ferdium => {
         if (parentNodeOfParentNode) {
           const unreadCounts = parentNodeOfParentNode.querySelectorAll('.bsU');
           if (unreadCounts.length > 0) {
-            count = Ferdium.safeParseInt(unreadCounts[0].textContent.replace(/[^\p{N}]/gu, ''));
+            let unreadCount = unreadCounts[0].textContent;
+            if (unreadCount.includes(':')) {
+              let counts = unreadCount
+                .split(':')
+                .map(s => Ferdium.safeParseInt(s.replace(/[^\p{N}]/gu, '')));
+              count = counts[0];
+              count2 = counts[1] - counts[0];
+            } else {
+              count = Ferdium.safeParseInt(
+                unreadCount.replace(/[^\p{N}]/gu, ''),
+              );
+            }
           }
         }
       }
     }
-
-    // set Ferdium badge
-    Ferdium.setBadge(count);
+    Ferdium.setBadge(count, count2);
```

Comment:
Have you tested this locally?

Reply:
yes i have tested this.

If there are important notification there number gets shown in a red bubble, if not but there is at least one not important notification a blue dot get shown.